### PR TITLE
Collection Tag in full page component

### PR DESCRIPTION
**Pull request by dblodorn**
Pretty straightforward

- if nft data has openseaInfo that is used for the contract name & icon - otherwise defaults to zoraNft.
- Placement is above title and below media by default - if we stick to this perhaps remove the top margin on the title h2 element?

Screen Shot 2021-10-18 at 5 16 36 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zoralabs/nft-components",
-  "version": "0.1.12",
+  "version": "0.1.15",
```
Comment (iainnash) [inline package.json:3]: nit: can this package change be removed so the version bump can be done independently of PRs

**Review by iainnash — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dblodorn) [reply package.json:3]: Sure.

Comment (dblodorn) [reply package.json:3]: Done @iainnash

Comment (iainnash) [reply package.json:3]: Looks like something went wrong here – can you try `git fetch origin main && git checkout origin/main -- package.json CHANGELOG`

Comment (dblodorn) [reply package.json]: Done @iainnash

**Pull request merged**